### PR TITLE
TypeScript root README

### PR DESCRIPTION
**Pull request by curtisman**
- Add a clear overview on how to get started.
- Update `getKey` description (and capabilities for change vault name)

### CODE REVIEW

File: ts/README.md
Comment (pcdeadeasy) [inline ts/README.md:1]: Suggestion: Right now we are two entry point the cli and shell. One with a light weight GUI experience vs console based experience, can we add something to that effect?

Comment (pcdeadeasy) [inline ts/README.md:1]: In my case I had to uninstall the older version of the Azure CLI to get az login to work correctly. Might be worth mentioning that.

Comment (curtisman) [reply ts/README.md:1]: Sound more like an az cli issue. How about I add latest to the previous line (Install Latest Azure CLI)

Comment (curtisman) [reply ts/README.md:1]: I think that we should focus the shell on first run. and provide the CLI as an alternative. later.  The CLI is mentioned in the "Running" section later.

Comment (curtisman) [reply ts/README.md:1]: I have expanded the running section a little bit.

**Pull request merged**